src/InverseKinematicsIPOPT.cpp

```diff
@@ -0,0 +1,499 @@
+#include "InverseKinematicsIPOPT.h"
+
+#include <cassert>
+
+using namespace std;
+using namespace iDynTree;
+using namespace Ipopt;
+
+InverseKinematicsIPOPT::InverseKinematicsIPOPT()
+: exitCode(-12)
+, parentFrame(0)
+, endEffectorFrame(0)
+,modelLoaded(false)
```

Comment:
be consistent with the use of whitespace characters. I prefer the space between comma and word